webclient/templates/toolbar_dokument.html

```diff
@@ -11,11 +11,11 @@
     </div>
     <div id="id-app-entity-item" class="app-entity-item">
       {% if d.ident_cely %}
-      {{ d.ident_cely }}{% if showback %}<div id="document-back" style="display:none"><span class="app-pipe"></span><a href="" class="app-entity-color font-weight-normal"></a></div>{% endif %}
-      {% elif index%}
-      {% else %}
-      {% if is_3d %}{% trans "templates.toolbarDokument.knihovna3D.zapsat.label" %}{% else %}{% trans "templates.toolbarDokument.dokument.zapsat.label" %}{% endif %}
-      {% endif %}
+        {{ d.ident_cely }}{% if showback %}<div id="document-back" style="display:none"><span class="app-pipe"></span><a href="" class="app-entity-color font-weight-normal"></a></div>{% endif %}
+        {% elif index%}
+        {% else %}
+        {% if is_3d %}{% trans "templates.toolbarDokument.knihovna3D.zapsat.label" %}{% else %}{% trans "templates.toolbarDokument.dokument.zapsat.label" %}{% endif %}
```

Comment:
@jnihnat Tohle jsem upravoval jinak kvůli tomu, že takto tam vždy v základu problikne "Zapsat" a až poté se zobrazí odpovídající popisek. To mi nepřijde moc šťastné. Proto jsem to dával raději do těch dílčích view přes JS a výchozí variantu chtěl nechat prázdnou...

Reply:
Ještě doplňuji, že pokud by se to mělo dít i v těch ostatních upravených případech, tak by to také nebylo úplně ideální.

Reply:
Uz chapu, u ostatnich by to malo fungovat dobre. Predelam i dokumenty aby to taky fungovalo lepe a nebylo to v js.

Reply:
Děkuji!

Reply:
hotovo